osu.Game/Storyboards/Drawables/DrawableStoryboardSample.cs

```diff
@@ -42,6 +42,14 @@ protected override void SkinChanged(ISkinSource skin, bool allowFallback)
             }
         }
 
+        protected override void SamplePlaybackDisabledChanged(ValueChangedEvent<bool> disabled)
+        {
+            if (!RequestedPlaying) return;
+
+            if (disabled.NewValue)
+                Stop();
```

Comment:
This is going to set `RequestedPlaying` to false. I guess this doesn't really matter too much, but something to be aware of.